src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -186,7 +185,7 @@ class ProfileView extends React.Component {
 
 function mapStateToProps(state) {
   return {
-    showDirectDeposit: !isProduction(state) && profileShowDirectDeposit(state),
+    showDirectDeposit: profileShowDirectDeposit(state),
```

Comment:
the `profile_show_direct_deposit` Flipper flag is completely off in prod so this won't actually release the feature yet.